pkgs/by-name/pr/prometheus-cpp/package.nix

```diff
@@ -41,11 +46,6 @@ stdenv.mkDerivation (finalAttrs: {
     "dev"
   ];
 
-  postInstall = ''
-    mkdir -p $dev/lib/pkgconfig
-    substituteAll ${./prometheus-cpp.pc.in} $dev/lib/pkgconfig/prometheus-cpp.pc
```

Comment:
I'm assuming this file can also be deleted now?

Reply:
That slipped my mind. Thanks for the catch!